src/components/Sum/Sum.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './sum.scss';
+
+const Sum = ({
+  firstNumber,
+  seconNumber,
```

Comment:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
```suggestion
  secondNumber,
```